php/README.md

```diff
@@ -0,0 +1,34 @@
+# HubSpot-php sample OAuth 2.0 app
+
+This is a sample app for the [hubspot-php SDK](../../../../). Currently, this app focuses on demonstrating the functionality of [OAuth API](https://developers.hubspot.com/docs-beta/working-with-oauth) endpoints and their related actions.
+
+Please see the documentation on [Creating an app in HubSpot](https://developers.hubspot.com/docs-beta/creating-an-app)
```

Comment:
Currently we have general README for all languages, README per language has only configure and installing instructions per language. Please, follow the format.

Reply:
I've just deleted readme
There is only code base